jenkins/ci.suse.de/cloud-mediacheck.yaml

```diff
@@ -109,7 +110,7 @@
           # only enable jtsync when build is not manually triggered
           if [[ "${ROOT_BUILD_CAUSE}" != "MANUALTRIGGER" ]]; then
             trap - EXIT ERR
-            $jtsync --ci suse --matrix ${project},${subproject},${BUILD_NUMBER} $RETVAL || :
+            $jtsync --ci suse --matrix ${name},${project}${subproject},${BUILD_NUMBER} $RETVAL || :
```

Comment:
is it intentional that it uses $subproject here, but not in jtsync_trap ?

Reply:
You are right. I can remove $subproject there, too. The variable $project should be enough to extract the version.